src/modules/report/services/kmsEncryption.ts

```diff
@@ -0,0 +1,57 @@
+import { KMSClient, EncryptCommand, DecryptCommand } from '@aws-sdk/client-kms'
+import { logger } from '../../../core/services/LoggerService'
+
+const kmsClient = new KMSClient({
+  region: process.env.AWS_REGION || 'us-east-1',
+})
+
+const kmsKeyId = process.env.AWS_KMS_KEY_ID
```

Comment:
Shouldn't reviewers of this PR have this environment variable defined in order to do peer testing? Also, shouldn't this be documented in the README?

I'm guessing that because I don't have this env variable, when I try to test your PR, it's just storing them as plain text still.